wormhole-connect/src/views/Redeem/Redeem.tsx

```diff
@@ -90,7 +94,14 @@ function Redeem({
       while (signed === undefined && !cancelled) {
         try {
           signed = await RouteOperator.getSignedMessage(route, txData);
-        } catch {
+        } catch (e: any) {
+          console.trace(e);
+          if (
+            e?.message === 'There are not enough valid signatures to repair.'
```

Comment:
minor comment but this could break if someone changes the error message slightly where it gets emitted. maybe make it a const?